From 103e6061d24eb7eaaafe1677b057fce99208a400 Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Mon, 28 Aug 2023 09:16:38 +0000
Subject: [PATCH] Enable kernel configs for vendor interface compatibility

VINTF compatibility check failing due to CONFIG_HID_PLAYSTATION and
CONFIG_PLAYSTATION_FF not enabled.

Fix the issue by enabling CONFIG_HID_PLAYSTATION, CONFIG_PLAYSTATION_FF
and its dependent drivers.

Tracked-On: OAM-110604
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 .../gmin64/config-lts/lts2022-chromium/x86_64_defconfig      | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/groups/kernel/gmin64/config-lts/lts2022-chromium/x86_64_defconfig b/groups/kernel/gmin64/config-lts/lts2022-chromium/x86_64_defconfig
index 8a3da96..8674994 100644
--- a/groups/kernel/gmin64/config-lts/lts2022-chromium/x86_64_defconfig
+++ b/groups/kernel/gmin64/config-lts/lts2022-chromium/x86_64_defconfig
@@ -5021,6 +5021,8 @@ CONFIG_HID_PICOLCD=y
 # CONFIG_HID_PICOLCD_LEDS is not set
 # CONFIG_HID_PICOLCD_CIR is not set
 CONFIG_HID_PLANTRONICS=y
+CONFIG_HID_PLAYSTATION=y
+CONFIG_PLAYSTATION_FF=y
 # CONFIG_HID_PXRC is not set
 # CONFIG_HID_RAZER is not set
 # CONFIG_HID_PRIMAX is not set
@@ -5462,7 +5464,7 @@ CONFIG_SCSI_UFSHCD_PCI=y
 CONFIG_NEW_LEDS=y
 CONFIG_LEDS_CLASS=y
 # CONFIG_LEDS_CLASS_FLASH is not set
-# CONFIG_LEDS_CLASS_MULTICOLOR is not set
+CONFIG_LEDS_CLASS_MULTICOLOR=y
 # CONFIG_LEDS_BRIGHTNESS_HW_CHANGED is not set
 
 #
@@ -5507,6 +5509,7 @@ CONFIG_LEDS_CLASS=y
 #
 # RGB LED drivers
 #
+# CONFIG_LEDS_PWM_MULTICOLOR is not set
 
 #
 # LED Triggers
-- 
2.39.2

